CHANGELOG.md

```diff
@@ -18,3 +18,8 @@
 
 - **New Feature**
   - Initial Release (@IMax153)
+
+# 0.1.1
```

Comment:
Since we are not pushing out a major version, and we have not yet hit the `v0.1.0` release, the middle number would be considered the major version number. Therefore, I would change this to `0.0.2`.

In addition, the `CHANGELOG` should proceed in descending order of entry date. So the most *recent* entries should occur first in the `CHANGELOG`.